recipes/libglu/bld.bat

```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+
+:: Configure.
```

Comment:
@ocefpaf I have no clue about the Windows build and the syntax. Are static builds supported? Should I just skip it in win?